javascripts/background_scripts/embeded.js

```diff
@@ -0,0 +1,38 @@
+/*global chrome */
```

Comment:
Needs file overview. Also, why is this called "embeded.js"? You previously labeled the other posting method as embeded.

I think we have been mispelling embedded for a while. embeded -> embedded